src/helix/client/client_ext.rs

```diff
@@ -901,22 +988,63 @@ impl<'client, C: crate::HttpClient + Sync + 'client> HelixClient<'client, C> {
         Ok(self.req_put(req, helix::EmptyBody, token).await?.data)
     }
 
-    /// Get multiple users chat colors
-    pub async fn get_users_chat_colors<T>(
+    /// Get a user's chat color
+    ///
+    /// [`None`](Option::None) is returned if the user never set their color in the settings.
+    pub async fn get_user_chat_color<T>(
         &'client self,
-        user_ids: impl AsRef<[&types::UserIdRef]> + Send,
+        user_id: impl Into<&types::UserIdRef> + Send,
         token: &T,
-    ) -> Result<Vec<helix::chat::UserChatColor>, ClientError<C>>
+    ) -> Result<Option<helix::chat::UserChatColor>, ClientError<C>>
     where
         T: TwitchToken + Send + Sync + ?Sized,
     {
-        let user_ids = user_ids.as_ref();
-        if user_ids.len() > 100 {
-            return Err(ClientRequestError::Custom("too many IDs, max 100".into()));
-        }
-        let req = helix::chat::GetUserChatColorRequest::user_ids(user_ids);
+        Ok(self
+            .req_get(
+                helix::chat::GetUserChatColorRequest::user_ids(&[user_id.into()][..]),
```

Comment:
```suggestion
                helix::chat::GetUserChatColorRequest::user_ids(&user_id.into()),
```

Not super important, but looks a bit simpler.